src/tools/genpolicy/rules.rego

```diff
@@ -675,27 +676,27 @@ allow_arg(i, i_arg, p_process, s_name) {
 }
 
 # OCI process.Env field
-allow_env(p_process, i_process, s_name) {
+allow_env(p_process, i_process, s_name, s_namespace) {
     print("allow_env: p env =", p_process.Env)
     print("allow_env: i env =", i_process.Env)
 
     every i_var in i_process.Env {
         print("allow_env: i_var =", i_var)
-        allow_var(p_process, i_process, i_var, s_name)
+        allow_var(p_process, i_process, i_var, s_name, s_namespace)
```

Comment:
Can we use a separate path, specific to 

s_namespace != ""

that uses s_namespace, instead of adding s_namespace as parameter to all the allow_var() variants?